lib/ansible/module_utils/powershell.ps1

```diff
@@ -205,6 +205,10 @@ Function Get-AnsibleParam($obj, $name, $default = $null, $resultobj = @{}, $fail
         if ($type -eq "path") {
             # Expand environment variables on path-type
             $value = Expand-Environment($value)
+            # Test if a valid path is provided
+            if (-not (Test-Path -IsValid $value)) {
```

Comment:
Good question, I refrained from making this modifications in other modules (where it should be -LiteralPath), but here it is uncertain. Since I am mostly interested in invalid paths (outside of the scope of wildcards) I don't mind changing it either way.

To be discussed.